Documentation/building/testing-with-corefx.md

```diff
@@ -34,7 +34,7 @@ Instead of copying CoreCLR binaries you can also test your changes with an exist
 
 ### Locally-built CoreFX 
 
-Once you have finished steps 1, 2. and 4. above execute the following instructions to test your local CLR changes with the built-CoreFX changes.
+Once you have finished steps 1, 2. and 4. above executing the following instructions to test your local CLR changes with the built-CoreFX changes.
```

Comment:
This is wrong; "execute" was correct. If a change is desired here, it should be to add a comma before "execute" and to remove the period after "4".